Update index.md

Mo-xiaoyu opened the pull request:
fix kbd

Mo-xiaoyu committed:
Update index.md

fix kbd

opbld33 commented:
Learn Build status updates of commit _[e5fb47d](https://github.com/Mo-xiaoyu/windows-dev-docs/commits/e5fb47d38857d8ad63f6eef45ed062d5e55d247c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[hub/powertoys/index.md](https://github.com/Mo-xiaoyu/windows-dev-docs/blob/patch-1/hub/powertoys/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8f0bf4e1-986f-a415-e316-169a990c8a5d/202211161511483032-4152/BuildReport?accessString=638b560c8affdfe87ca165fd9686cf5343cade78e29c0bed111f4dee4d4c795e).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8f0bf4e1-986f-a415-e316-169a990c8a5d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

mattwojo reviewed (approved):
#sign-off

Mo-xiaoyu commented:
> @Mo-xiaoyu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

You're welcome

mattwojo committed:
Replace "just" 

...generally try to avoid the term "Just" do this or that as recommended by our writing guidance in favor of more explicit language.

opbld33 commented:
Learn Build status updates of commit _[8f89ff6](https://github.com/Mo-xiaoyu/windows-dev-docs/commits/8f89ff633e04e9bef96d3f80045892966fb34276)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[hub/powertoys/index.md](https://github.com/Mo-xiaoyu/windows-dev-docs/blob/patch-1/hub/powertoys/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8f0bf4e1-986f-a415-e316-169a990c8a5d/202211172127040498-4152/BuildReport?accessString=219f8099c983ca3b0e4c2c512426bc22018eb7cf5443c94d1302cad29e054f4f).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8f0bf4e1-986f-a415-e316-169a990c8a5d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>
